instrumentation/opentelemetry-instrumentation-httpx/tests/test_httpx_integration.py

```diff
@@ -725,7 +725,6 @@ def test_client_mounts_with_instrumented_transport(self):
                 spans[1].attributes[SpanAttributes.HTTP_URL], https_url
             )
 
-    @mock.patch.dict("os.environ", {"NO_PROXY": ""}, clear=True)
```

Comment:
This is useful for having tests separated from the environment in which they are run

Reply:
Right. I missed that we can have conflicts with the local environment variables from the host